pkg/kubelet/metrics/collectors/resource_metrics_test.go

```diff
@@ -85,6 +92,9 @@ func TestCollectResourceMetrics(t *testing.T) {
 				# HELP node_memory_working_set_bytes [ALPHA] Current working set of the node in bytes
 				# TYPE node_memory_working_set_bytes gauge
 				node_memory_working_set_bytes 1000 1624396278302
+				# HELP node_swap_usage_non_windows_bytes [ALPHA] Current swap usage of the node in bytes. Reported only on non-windows systems
```

Comment:
hm I am not a huge fan of `non_windows` here. is there a convention set that says we should add that?